google-datacatalog-rdbms-connector/tests/google/datacatalog_connectors/rdbms/test_data/sql_objects_ingest_cfg.yaml

```diff
@@ -0,0 +1,5 @@
+sql_objects:
```

Comment:
As per one of my previous comments, doesn't it make sense to add this content to the existing `ingest_cfg.yaml` file?

Reply:
I have different testing  scenarios, that need to test the separate config.

I have updated the docs to not point to the test data,  and on the following PR it will point to a real use case of this mechanism.